README.md

```diff
@@ -673,7 +678,7 @@ const previousEpochObservers = await arIO.getPrescribedObservers({
   {
     "gatewayAddress": "2Ic0ZIpt85tjiVRaD_qoTSo9jgT7w0rbf4puSTRidcU",
     "observerAddress": "2Ic0ZIpt85tjiVRaD_qoTSo9jgT7w0rbf4puSTRidcU",
-    "stake": 10000,
+    "stake": 10000, // vault in mIO
```

Comment:
Should maybe update this readme example as I think this wouldn't be a typical amount of mIO for operatorStake